boot/bootutil/include/bootutil/bootutil_log.h

```diff
@@ -135,16 +135,21 @@ int sim_log_enabled(int level);
  */
 #elif defined(MCUBOOT_MYNEWT)
 
+#include <mcuboot_config/mcuboot_config.h>
 #include <stdio.h>
 
-#define BOOT_LOG_LEVEL_OFF      0
-#define BOOT_LOG_LEVEL_ERROR    1
-#define BOOT_LOG_LEVEL_WARNING  2
-#define BOOT_LOG_LEVEL_INFO     3
-#define BOOT_LOG_LEVEL_DEBUG    4
+#define BOOT_LOG_LEVEL_OFF      1
+#define BOOT_LOG_LEVEL_ERROR    2
+#define BOOT_LOG_LEVEL_WARNING  3
+#define BOOT_LOG_LEVEL_INFO     4
+#define BOOT_LOG_LEVEL_DEBUG    5
 
 #ifndef BOOT_LOG_LEVEL
-#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_INFO
+#define BOOT_LOG_LEVEL MCUBOOT_LOG_LEVEL
```

Comment:
It seems like what happened is that a patch was added on top of the existing series which moved it closer to what I was proposing.

However, this series still has the problems I named, and what I meant by "re-work" was that the patches themselves would be changed using something like an interactive git rebase so that the problems would be avoided by structuring the series in a different way.

For example, this patch still has the same problems:

- introduces MCUBOOT_LOG_LEVEL, apparently to be obtained from mcuboot_config.h, without documenting it in the template file
- values of this macro still depend on other values that don't come from the config file

That's not what I had proposed -- in my proposal, that the only addition to mcuboot_config.h would be MCUBOOT_HAVE_LOGGING. The log level defines (and MCUBOOT_LOG_LEVEL) would be handled by mcuboot_logging.h when it existed. This avoids the problems named earlier.